semeio/jobs/scripts/fm_pyscal.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+"""Forward model for connecting ERT with the Pyscal command line client"""
+import logging
+import sys
+import os
+import six
+
+from semeio.jobs.design_kw.design_kw import extract_key_value
+
+from pyscal import pyscalcli
+
+_logger = logging.getLogger("FM_PYSCAL")
```

Comment:
What is this logger? If the logger is not defined, will it default to standard streams?

Reply:
This line was stolen from `fm_stea.py`, except for the explicit naming. It ends in stderr by default (so it seems).